packages/analyzer/src/polymer/polymer-element.ts

```diff
@@ -150,14 +150,14 @@ export interface Options {
   observers: Observer[];
   listeners: {event: string, handler: string}[];
   behaviors: ScannedReference<'behavior'>[];
-
   events: Map<string, ScannedEvent>;
-
   abstract: boolean;
   privacy: Privacy;
   astNode: AstNodeWithLanguage;
   statementAst: babel.Statement|undefined;
   sourceRange: SourceRange|undefined;
+  /** Whether this element was defined using the legacy Polymer() function. */
```

Comment:
Nit, slightly clearer as: `true iff this element was defined using the legacy Polymer() function.`

Reply:
Done.